Code_Exercises/Exercise_09_Synchronization/solution.cpp

```diff
@@ -50,9 +50,11 @@ TEST_CASE("buffer_accessor_event_wait", "synchronization_solution") {
         std::rethrow_exception(e);
       }
     };
-
-    auto defaultQueue = sycl::queue{sycl::default_selector_v, asyncHandler};
-
+#if defined(SYCL_LANGUAGE_VERSION) && defined(__INTEL_LLVM_COMPILER)
+    auto defaultQueue = sycl::queue{sycl::default_selector{}, asyncHandler};
+#else
+	auto defaultQueue = sycl::queue{sycl::default_selector_v, asyncHandler};
```

Comment:
The formatting is different in this branch. But I think that we could just use the first branch for all compilers because it's still valid SYCL 2020. The same for other places.